### PR TITLE
BaSP-TG-76: Employee My Projects screen

### PR DESCRIPTION
**Pull request by FranGutierrez**
- Created routes for Employee screens
- Created Employee My Projects screen to show employee's assigned projects
- Modified the Layout component to show the correct sidebar according the routes
- Modified the list shared component and the implementations to show edit & delete buttons if necessary

### CODE REVIEW

File: src/redux/employees/thunks.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ import {
   deleteEmployeesError,
   resetEmployee,
   resetMessage
-} from './actions.js';
+} from './actions';
 
 const resource = `${process.env.REACT_APP_API_URL}/employees`;
```
Comment (Julian25) [inline src/redux/employees/thunks.js:21]: cambiar resource por url

**Review by Julian25 — APPROVED** [review]

**Review by MatiasJoaquinReynoso — APPROVED** [review]
All works correctly !

File: src/routes/index.jsx [hunk 2]
```diff
@@ -41,6 +42,10 @@ const Routes = () => {
             <Route exact path={'/timesheets'} component={TimeSheetsList} />
             <Route exact path={'/timesheets/form/'} component={TimeSheetsForm} />
             <Route exact path={'/timesheets/form/:id'} component={TimeSheetsForm} />
+            <Route exact path={'/employee'} />
+            <Route exact path={'/employee/projects'} component={EmployeeProjects} />
+            <Route exact path={'/employee/profile'} />
+            <Route exact path={'/employee/workedhours'} />
```
Comment (BaltazarCortina1) [inline src/routes/index.jsx:48]: Aca deberias haber llamado a employeeRoutes, con la ruta /employee

File: src/routes/employees.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React, { lazy } from 'react';
+import { BrowserRouter as Route } from 'react-router-dom';
+const EmployeeProjects = lazy(() => import('Components/Employee/MyProjects'));
```
Comment (BaltazarCortina1) [inline src/routes/employees.jsx:3]: Dentro de este archivo no tiene mucho sentido usar lazy. Podemos importar todo directamente

**Pull request merged**